src/components/resumes/__tests__/ResumeList.test.js

```diff
@@ -0,0 +1,52 @@
+import Resumes from "../Resumes";
+import axios from "axios";
+import { getResumes } from "../ResumesAPI.js";
+import { resumes } from "../fixtures";
+import { MemoryRouter, Route } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+jest.mock("axios");
+
+const API_HOST = process.env.REACT_APP_API_RESUMES_URL;
+const API_NAME = "resumes";
+const API_VERSION = "v1";
+const url = `${API_HOST}/api/${API_VERSION}/${API_NAME}`;
+
+test("should return an empty list", async () => {
+  axios.get.mockImplementationOnce(() => Promise.resolve({ data: [] }));
+
+  await expect(getResumes()).resolves.toEqual([]);
+});
```

Comment:
remove this we don't neeed to test axios

Reply:
@DaleneyRuben done :D